src/routes/auth/magic-url/+page.svelte

```diff
@@ -4,6 +4,7 @@
     import { Heading } from '$lib/components';
     import { Account, Client } from '@appwrite.io/console';
     import { onMount } from 'svelte';
+    import LL from '../../../i18n/i18n-svelte';
```

Comment:
An alias would be nice https://kit.svelte.dev/docs/configuration#alias

Reply:
changes made.